user-stories/US-07-Data-rights-request-capture-service.md

```diff
@@ -154,3 +154,25 @@
 **I want to** use blindnet devkit to automate the processing of requests except for "other" type of request
 
 **because** I need to process "other" type of request manually 
+
+### US-07-3-07
+**As a** developper from an organization, using blindnet devkit
+
+**I want to** use only the capture interface component (from the HLA) to generate a json file for a request
+
+**because** this all I need, as I want to handle the file and import it to some other system 
```

Comment:
```suggestion
**because** this is all I need, as I want to handle the file and import it to some other system 
```

Reply:
Done (manually, I don't know why when I accepted your suggestion github said "File not found in repository. Refresh and try again")